lib/rust/parser/src/syntax/tree.rs

```diff
@@ -990,7 +1003,7 @@ pub fn to_ast(token: Token) -> Tree {
             Tree::text_literal(default(), default(), vec![newline], default(), default())
         }
         token::Variant::Wildcard(wildcard) => Tree::wildcard(token.with_variant(wildcard), default()),
-        token::Variant::AutoScope(t) => Tree::auto_scope(token.with_variant(t)),
+        token::Variant::SuspendedDefaultArguments(t) => Tree::suspended_default_arguments(token.with_variant(t)),
```

Comment:
Why do you call it `SuspendedDefaultArguments`? As far as I know there can be _suspended argument_ without any default.